src/Manager.php

```diff
@@ -218,16 +222,21 @@ public function updateRole(string $name, Role $role): self
         return $this;
     }
 
+    public function removeRole(string $name): self
+    {
+        $this->removeItem($name);
+        return $this;
+    }
+
     public function addPermission(Permission $permission): self
     {
         $this->addItem($permission);
         return $this;
     }
 
-    public function removePermission(string $permissionName): self
+    public function getPermission(string $name): ?Permission
     {
-        $this->removeItem($permissionName);
-        return $this;
+        return $this->itemsStorage->getPermission($name);
```

Comment:
This method may returns role instead of permission. This will result in a fatal error.